PowerShell/README.md

```diff
@@ -0,0 +1,52 @@
+# BitTitan PowerShell
+This folder contains a set of tools, building blocks and full scenario scripts tailored to work with the BitTitan PowerShell SDK. 
+
+## Quick Startup
+1. Install the [BitTitan PowerShell SDK](https://www.bittitan.com/downloads/bittitanpowershellsetup.msi).
+2. Clone the [bittitan-sdk repository](https://github.com/BitTitan/bittitan-sdk) or download the desired scripts.
+    ```shell
+    git clone https://github.com/BitTitan/bittitan-sdk.git
+    ```
+3. Move the `Init.ps1` (bittitan-sdk/PowerShell/Tools/Init.ps1) script to the working directory. If the repository was not cloned, download the [Init script](https://github.com/BitTitan/bittitan-sdk/blob/master/PowerShell/Tools/Init.ps1) and move it to the working directory.
```

Comment:
Copy the `Init.ps1` instead of Move?

Reply:
The concern is either moving or copying Init.ps1 is that it will get stale. We can discuss more elaborate guidance where Init.ps1 is available in the console's path so that the following just works:

* Assuming BitTitan SDK is under C:\Program Files\...
* Assuming customer's script is under user's folder like c:\users\joe\Document\PowerShell\MSPC or C:\Temp
* Customer can just do the following and the latest copy of init.ps1 will still be used
  cd c:\temp\
  .\my-script.ps1

Reply:
Sure, we can discuss this. For now, I've changed "move" to "copy".